evgen/array.hash.rb

```diff
@@ -0,0 +1,42 @@
+
+#2
+names.map(&:size).sum
+
+print names.each { |el| print el.length }
+
+count = 0
+names.each {|s| count += s.length}
+puts count
+
+#3
+names = names.map(&:downcase)
+
+#4
+hash = {}
+hash = names.each_with_object({}) do|element, object| object[element] = element.size
```

Comment:
Separate the code of your block with a newline or write it all in one line
```suggestion
hash = names.each_with_object({}) do |element, object|
  object[element] = element.size
end
```
or
```suggestion
hash = names.each_with_object({}) { |element, object| object[element] = element.size }
```